modules/stochastic_tools/include/state_sim/model/actions/StateSimAction.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMACTION_H
+#define STATESIMACTION_H
+
+#include "StateSimBase.h"
+#include <string>
+
+class StateSimAction : public StateSimBase
+{
+protected:
+  ACTION_TYPE_ENUM _act_type;
+
+private:
+  static unsigned int _counter;
```

Comment:
Sorry I totally read that wrong.  To much PRA on my brain.  Can I just declare it std::atomic_int then?